<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditDriver</class>
 <widget class="QDialog" name="EditDriver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Driver</string>
  </property>
  <widget class="QLineEdit" name="nameInput">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QListWidget" name="driverList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>191</width>
     <height>101</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="removeDriver">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove selected</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Driver List</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addDriver">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add driver</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeDriver">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
